packages/app/src/components/DimensionsPanel/DndDimensionItem.js

```diff
@@ -20,6 +20,7 @@ export class DndDimensionItem extends Component {
         } = this.props
 
         const itemCommonProps = {
+            id,
```

Comment:
Wouldn't this result in the same id being used in both the original and the cloned `DimensionItem` elements?

Reply:
Nevermind, I just read your explanation in the description.